HISTORY.md

```diff
@@ -21,6 +21,7 @@ schedulers.
 #### Documentation
 
 - Add BibTex citation for GECCO 2023 (#337)
+- Use dask instead of multiprocessing for lunar lander tutorial (#346)
```

Comment:
Can you make a new section called `(Upcoming)` and put this in a new Changelog section under that? This change will be part of our next release (0.5.2 is our latest)